bin/update_submodules.sh

```diff
@@ -4,9 +4,10 @@
 git submodule update --init --recursive
 
 # update monero-cpp
+# needs to be uncommented and changed to a release tag when release 
 cd ./external/monero-cpp
-git checkout tags/v0.3.7
-git pull --ff-only origin tags/v0.3.7
+#git checkout tags/v0.3.7
+#git pull --ff-only origin tags/v0.3.7
```

Comment:
Please uncomment and change `tags/v0.3.7` to `master` for these lines.  Release tags will checkout a specific version.

Reply:
I got confused. These changes don't make sense :/